AdminWebsite/AdminWebsite/ClientApp/src/app/app.module.ts

```diff
@@ -94,7 +94,7 @@ export function getSettings(configService: ConfigService) {
         ErrorService,
         PageTrackerService,
         AppInsightsLogger,
-        WindowRef
+        WindowRef,
```

Comment:
You might want to check you're using the same linting or prettier rules. I didn't expect this to change.

Reply:
Good point, will remove